ibm_catalog.json

```diff
@@ -92,6 +92,27 @@
                             }
                           ]
                         },
+                        {
```

Comment:
the order of the variables here matter, as thats the order they show in the UI. We try to group related variables together. You after adding this between `kms_endpoint_type` and `scc_cos_key_ring_name` which are both KMS related vars. Can you group the attachment related variables together. You also need to add `resource_groups_scope` too so it gets ordered